src/types.ts

```diff
@@ -1052,50 +1053,7 @@ export type EventHandler<
   UserType extends UR = UR
 > = (event: Event<AttachmentType, ChannelType, CommandType, EventType, MessageType, ReactionType, UserType>) => void;
 
-export type EventTypes =
-  | 'all'
-  | 'channel.created'
-  | 'channel.deleted'
-  | 'channel.hidden'
-  | 'channel.muted'
-  | 'channel.truncated'
-  | 'channel.unmuted'
-  | 'channel.updated'
-  | 'channel.visible'
-  | 'transport.changed' // ws vs longpoll
-  | 'connection.changed'
-  | 'connection.recovered'
-  | 'health.check'
-  | 'member.added'
-  | 'member.removed'
-  | 'member.updated'
-  | 'message.deleted'
-  | 'message.new'
-  | 'message.read'
-  | 'message.updated'
-  | 'notification.added_to_channel'
-  | 'notification.channel_deleted'
-  | 'notification.channel_mutes_updated'
-  | 'notification.channel_truncated'
-  | 'notification.invite_accepted'
-  | 'notification.invite_rejected'
-  | 'notification.invited'
-  | 'notification.mark_read'
-  | 'notification.message_new'
-  | 'notification.mutes_updated'
-  | 'notification.removed_from_channel'
-  | 'reaction.deleted'
-  | 'reaction.new'
-  | 'reaction.updated'
-  | 'typing.start'
-  | 'typing.stop'
-  | 'user.banned'
-  | 'user.deleted'
-  | 'user.presence.changed'
-  | 'user.unbanned'
-  | 'user.updated'
-  | 'user.watching.start'
-  | 'user.watching.stop';
```

Comment:
this list is bigger than the other

Reply:
This list is smaller actually than the other. Its missing `channel.kicked` event, which is fixed by this change

Reply:
This one has 42 types (excluding `all`), and other one has 43.

Reply:
right, it was diff not expanded completely. The place of event might be local ones though

Reply:
otherwise, good reduction in duplication